contrib/resources/translations/it.lng

```diff
@@ -1373,12 +1400,13 @@ Inoltre è possibile ridimensionare la finestra dopo l'avvio dell'emulatore.
 Imposta la posizione iniziale della finestra per la modalità finestra:
   auto:      Lascia che sia il gestore delle finestre a decidere la posizione
              (predefinito).
-  X,Y:       Imposta la posizione della finestra nel formato X,Y
-             (ad esempio 250,100).
+  X,Y:       Imposta la posizione della finestra nel formato X,Y in unità
```

Comment:
Non ho idea di cosa siano le unità logiche 😅

Reply:
Nel dubbio l'ho rimosso. Non capisco il significato del termine `unità logiche` in questo contesto.